### ISSUE TITLE
Set up php_cs-fixer for Travis-CI

### ISSUE THREAD

**Issue by sergey-shandar** [Azure/azure-sdk-for-php#896]

**Comment by sergey-shandar** [comment]
See also
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PSR2/ruleset.xml

**Comment by alcohol** [comment]
Why not https://github.com/FriendsOfPHP/PHP-CS-Fixer ? PHP CodeSniffer is rather outdated.

**Comment by sergey-shandar** [comment]
Yes, we already use it https://github.com/Azure/azure-sdk-for-php/blob/master/composer.json

**Comment by sergey-shandar** [comment]
But you are right, we need to remove squizlabs/php_codesniffer
